berkeley-function-call-leaderboard/bfcl_eval/eval_checker/eval_runner.py

```diff
@@ -33,8 +34,14 @@ def get_handler(model_name):
 
 
 def multi_turn_runner(
-    handler, model_result, prompt, possible_answer, model_name, test_category, score_dir
-):
+    handler: str,
```

Comment:
handler should be of type `BaseHandler` (from `berkeley-function-call-leaderboard/bfcl_eval/model_handler/base_handler.py`)